common/src/main/java/com/wynntils/features/user/overlays/CustomBarsOverlayFeature.java

```diff
@@ -183,6 +209,67 @@ public void render(PoseStack poseStack, float partialTicks, Window window) {
         }
     }
 
+    public static class AwakenedProgressBarOverlay extends HealthBarOverlay {
```

Comment:
No reason to inherit HealthBarOverlay. Now, it just messed up configs (textColor), and added an unused method for the base class. Make a new base class and use that.